### PR TITLE
Task/fd 3119 update devise google authenticator to make it possible to work with rails 5

### PR DESCRIPTION
**Pull request by ritaly**
let's get ready for Rails 5

### CODE REVIEW

File: app/controllers/devise/checkga_controller.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def update
         warden.manager._run_callbacks(:after_set_user, resource, warden, {:event => :authentication})
         respond_with resource, :location => after_sign_in_path_for(resource)
 
-        if not resource.class.ga_remembertime.nil? 
+        if !resource.class.ga_remembertime.nil? && (!resource.class.ga_remember_optional || Devise::TRUE_VALUES.include?(params['remember_me']))
```
Comment (MSzweda) [inline app/controllers/devise/checkga_controller.rb]: loooong liiine
break, possibly extract to named method

File: lib/devise_google_authenticatable/models/google_authenticatable.rb [hunk 2]
```diff
@@ -80,19 +80,35 @@ def first_authentication_key
           public_send(self.class.authentication_keys.first)
         end
 
+        def skip_validation? request
+          if self.class.ga_skip_validation_if.is_a? Proc
```
Comment (MSzweda) [inline lib/devise_google_authenticatable/models/google_authenticatable.rb]: why is this required O.o

**Comment by MSzweda** [comment]
this is reflecting changes from upstream

**Comment by dominika** [comment]
is this version working fine with current gaming_admin @ritaly ?
